niworkflows/utils/bids.py

```diff
@@ -145,6 +145,7 @@ def collect_participants(
 def collect_data(
     bids_dir,
     participant_label,
+    session_id=False,
```

Comment:
since 
```
session = None
```
is a valid way to differentiate within pybids, this defaults to `False`, which drops the `session` entity from the query entirely.

Reply:
What about using PyBIDS query semantics?

```suggestion
    session_id=bids.layout.Query.ANY,
```